File: Kotlin/bee 1035.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val (a, b, c, d) = br.readLine().split(" ").map {it.toInt()}

    if(b > c && d > a && c + d > a + b && c > 0 && d > 0 && a % 2 == 0){
        println("Valores aceitos")
    } else{
        println("Valores nao aceitos")
    }


}

File: Ruby/bee 1035.rb
a,b,c,d = gets.split(" ").map { |n| n.to_i }

if b > c && d > a && c + d > a + b && c > 0 && b > 0 && a % 2 == 0
  puts "Valores aceitos"
else
  puts "Valores nao aceitos"
end

